global.html

```diff
@@ -0,0 +1,134 @@
+<!DOCTYPE html>
```

Comment:
I love this.
![image](https://user-images.githubusercontent.com/60205816/182981303-a37c46db-482d-4e7b-a5ea-3cab21f11e79.png)